pkgs/applications/science/robotics/emuflight-configurator/default.nix

```diff
@@ -0,0 +1,50 @@
+{lib, stdenv, fetchurl, unzip, makeDesktopItem, nwjs, wrapGAppsHook, gsettings-desktop-schemas, gtk3 }:
+
+let
+  pname = "emuflight-configurator";
+  desktopItem = makeDesktopItem {
+    name = pname;
+    exec = pname;
+    icon = pname;
+    comment = "Emuflight configuration tool";
+    desktopName = "Emuflight Configurator";
+    genericName = "Flight controller configuration tool";
+  };
+in
+stdenv.mkDerivation rec {
+  inherit pname;
+  version = "0.3.5";
+  src = fetchurl {
+    url = "https://github.com/emuflight/EmuConfigurator/releases/download/${version}/emuflight-configurator_${version}_linux64.zip";
```

Comment:
Why are we not building from source?

Reply:
I could not get a source build to work.  The build requires a variety of node packages.  Specifically, I couldn't get node2nix to store the gulp dependency.  I am open suggestions if someone wants to help me to get it working though.  I don't know much about nodejs or nix expressions.

Reply:
I ended up using https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/science/robotics/betaflight-configurator/ as a template, emuflight-configurator is a fork of the betaflight-configurator.

Reply:
Maybe it is good that we do not store another node2nix file in the repo. They are always pretty big.